File: src/main/java/trap/model/RoundTotal.java
package trap.model;

public record RoundTotal(
        int eventId,
        int locationId,
        String team,
        String athlete,
        String classification,
        String gender,
        int total,
        String type
) {
    public String uniqueName() {
        return athlete + " " + team + " " + classification + " " + type;
    }
}

File: src/main/java/trap/model/RoundTotal.kt
package trap.model

class RoundTotal(val eventId: Int,
                 val locationId: Int,
                 val team: String? = null,
                 val athlete: String? = null,
                 val classification: String? = null,
                 val gender: String? = null,
                 val total: Int,
                 val type: String? = null) {

    val uniqueName: String
        get() = "$athlete $team $classification $type"
}
